RetrospectiveExtension.Backend/deploy/env_setup.sh

```diff
@@ -79,6 +79,21 @@
         --query instrumentationKey \
         --output tsv)
 
+    #Add an Application Insights dashboard
+    dashboard_file="./deploy/ai-dashboard.json"
+
+    cp ./deploy/ai-dashboard.json.template ${dashboard_file}
+  
+    sed -i "s/VARRGNAME/${resource_group}/"                     ${dashboard_file}
+    sed -i "s/VARSUBSCRIPTIONID/${subscription_id}/"            ${dashboard_file}
+    sed -i "s/VARAIRESOURCENAME/ai-${resource_name_suffix}/"    ${dashboard_file}
+
+    #https://docs.microsoft.com/en-us/cli/azure/portal/dashboard?view=azure-cli-latest
+    az portal dashboard create --location "eastus" \
+        --name "${resource_name_suffix}-dashboard" \
```

Comment:
```suggestion
        --name "dashboard-${resource_name_suffix}" \
```
@mhassaninmsft Switching to actual suffix to align with other service naming.